routes/web.php

```diff
@@ -290,6 +290,9 @@
     Route::post('session', 'SessionsController@store')->name('login');
     Route::delete('session', 'SessionsController@destroy')->name('logout');
 
+    Route::post('user-cover-presets/batch-activate', 'UserCoverPresetsController@batchActivate')->name('user-cover-presets.batch-activate');
+    Route::resource('user-cover-presets', 'UserCoverPresetsController', ['only' => ['index', 'store', 'update']]);
+
```

Comment:
shouldn't these go under `/admin`?

Reply:
not if we're going to allow this to be accessible by non-admin

Reply:
would be very nice to have it under `/admin` now in the console page, highly doubt any other usergroup will get access to this.

(another side effect is me having to look back at this PR each time to remember the route lol)